cluster/nginx.go

```diff
@@ -228,15 +229,59 @@ func UpdateNginxConfiguration(ctx *Context) chan error {
 	ch := make(chan error)
 	go func() {
 		defer close(ch)
-		tenantRoutes := <-GetAllTenantRoutes(ctx)
+
 		// creates the clientset
 		clientset, err := k8sClient()
 		if err != nil {
 			ch <- err
 			return
 		}
-		var nginxConfiguration bytes.Buffer
-		nginxConfiguration.WriteString(`
+
+		var nginxConfiguration string
+		// check whether global buckets are enabled
+		globalBuckets, err := GetConfig(nil, cfgCoreGlobalBuckets, false)
+		if err != nil {
+			ch <- err
+			return
+		}
+		if globalBuckets.ValBool() {
+			nginxConfiguration = getGlobalBucketNamespaceConfiguration()
+		} else {
+			nginxConfiguration = getLocalBucketNamespaceConfiguration(ctx)
+		}
+
+		fmt.Println(nginxConfiguration)
```

Comment:
Please change to `log.` if this is for debugging purposes

Reply:
forgot to remove this line